vitest.config.ts

```diff
@@ -1,9 +1,17 @@
 import { sveltekit } from '@sveltejs/kit/vite';
 import { svelteTesting } from '@testing-library/svelte/vite';
+import { readFileSync } from 'node:fs';
+import { fileURLToPath } from 'node:url';
 import { resolve } from 'path';
 import type { UserConfig } from 'vite';
 import { defineConfig } from 'vitest/config';
 
+const file = fileURLToPath(new URL('package.json', import.meta.url));
```

Comment:
In order to avoid code duplication, I suggest to rather move the common code in existing `vite.utils.ts`, expose a const or function within the utils that returns the data and use it in both `vite.config` and `vitest.config`.

Reply:
For example

```
export const defineReplacements = (): { VITE_APP_VERSION: string; VITE_DFX_NETWORK: string } => {
	const file = fileURLToPath(new URL('package.json', import.meta.url));
	const json = readFileSync(file, 'utf8');
	const { version } = JSON.parse(json);

	// npm run dev = local
	// npm run build = local
	// dfx deploy = local
	// dfx deploy --network ic = ic
	// dfx deploy --network staging = staging
	const network = process.env.DFX_NETWORK ?? 'local';

	return {
		VITE_APP_VERSION: JSON.stringify(version),
		VITE_DFX_NETWORK: JSON.stringify(network)
	};
};
```

Reply:
I am not completely convinced about `VITE_DFX_NETWORK` since theoretically for VITEST should be always local. Or not?